speed-dreams/source-2.2.3/src/modules/graphic/ssggraph/grboard.h

```diff
@@ -45,29 +45,31 @@ class cGrBoard
     int rightAnchor;
     int speedoRise;
 
- private:
-    void grDispDebug(const tSituation *s, const cGrFrameInfo* frame);
+private:
+    void grDispDebug(const tSituation *s, const cGrFrameInfo *frame);
     void grDispSplitScreenIndicator();
     void grDrawGauge(tdble X1, tdble Y1, tdble H, float *clr1,
-                        float *clr2, tdble val, const char *title);
+                     float *clr2, tdble val, const char *title);
 
     void grDispCounterBoard2();
 
     // SIMULATED DRIVING ASSISTANCE: add display indicator methods
-    void DispIndicators();
-    void DispIndicatorIcon(tTextureData* p_data, ssgSimpleState* p_texture);
-    void DispIndicatorText(tTextData* p_data);
+    void DispActiveIndicators(tIndicator settings, InterventionType interventionType);
+    void DispNeutralIndicators(tIndicator settings, InterventionType interventionType);
+    void DispIndicatorsHelper(tIndicatorData m_indicator, tIndicator settings);
+    void DispIndicatorIcon(tTextureData *p_data, ssgSimpleState *p_texture);
+    void DispIndicatorText(tTextData *p_data);
```

Comment:
This is one of our added functions, and thus should follow our coding standard, so undo the change moving the *